src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -1603,3 +1604,29 @@ def set_dataio(cls, **kwargs):
             data = H5DataIO(data)
         """
         return H5DataIO.__init__(**kwargs)
+
+    @staticmethod
+    def generate_dataset_html(dataset):
+        """Generates an html representation for a dataset for the HDF5IO class"""
```

Comment:
Would be useful to raise a corresponding issue on HDMF_ZARR to have `generate_dataset_html` be implemented on `ZarrIO` as well (if we have not done this yet). @stephprince can you make and issue